package.json

```diff
@@ -1,24 +1,24 @@
-{ 
+{
   "name": "spectral-workbench",
   "version": "1.0.0",
+  "private": "true",
   "repository": "https://github.com/publiclab/spectral-workbench/",
   "author": "Public Lab contributors <web@publiclab.org>",
-  "license": "GPL3",
+  "license": "GPL-3.0",
   "dependencies": {
     "ace-builds": "jywarren/ace-builds#v1.2.4",
     "bootstrap-css": "jozefizso/bower-bootstrap-css#~2.3.2",
-    "d3": "mbostock-bower/d3-bower#~3.3.13",
-    "flot": "flot/flot#~0.7",
-    "fontawesome": "FortAwesome/Font-Awesome#~4.5.0",
-    "getusermedia.js": "addyosmani/getUserMedia.js#v1.0rc2",
-    "jasmine": "pivotal/jasmine#~2",
-    "jasmine-ajax": "jasmine/jasmine-ajax#v3.2.0",
+    "d3": "^3.5.4",
+    "flot": "~0.8.0-alpha",
```

Comment:
This looks great. The graphs work on all the prior interfaces? Great work!

Reply:
Yes, I've checked locally as well as on gitpod, the live graph works as expected at the `/capture` route